SportsHub/SportsHub.Application/Authentication/PasswordHasher/PasswordHasher.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Extensions.Options;
+using SportsHub.AppService.Authentication.Models.Options;
+using SportsHub.Domain.PasswordHasher;
+using System.Security.Cryptography;
+
+namespace SportsHub.AppService.Authentication.PasswordHasher
+{ 
+    public sealed class PasswordHasher : IPasswordHasher
+    {
+        private const int SaltSize = 16; // 128 bit 
```

Comment:
Fields in the rest of the project are camelCase - maybe change the ones in this pull request of consistency?

Reply:
The properties are now renamed.